### ISSUE TITLE
travis t2006 failure

### ISSUE THREAD

**Issue by nigoroll** [varnishcache/varnish-cache#2293]
https://travis-ci.org/varnishcache/varnish-cache/jobs/216681470

```
**   s1    0.7 === txresp -hdr "Content-Type: text/plain" -body response
**** s1    0.7 txresp|HTTP/1.1 200 OK\r
**** s1    0.7 txresp|Content-Type: text/plain\r
**** s1    0.7 txresp|Content-Length: 8\r
**** s1    0.7 txresp|\r
**** s1    0.7 txresp|response
***  s1    0.7 shutting fd 5
**   s1    0.7 Ending
***  c1    0.7 rx: stream: 1, type: HEADERS (1), flags: 0x04, size: 106
***  c1    0.7 flag: END_TYPE_HEADERS
**** c1    0.7 header[ 0]: :status : 200
**** c1    0.7 header[ 1]: content-type : text/plain
**** c1    0.7 header[ 2]: content-length : 8
**** c1    0.7 header[ 3]: date : Thu, 30 Mar 2017 09:21:57 GMT
**** c1    0.7 header[ 4]: x-varnish : 1001
**** c1    0.7 header[ 5]: age : 0
**** c1    0.7 header[ 6]: via : 1.1 varnish (Varnish/5.1)
**** c1    0.7 header[ 7]: accept-ranges : bytes
---- c1    0.7 Wrong frame type HEADERS (1) wanted WINDOW_UPDATE

```

**Comment by bsdphk** [comment]
Fixed by b8d06876da481ce6e69ee2165d86e2118eabf915 and/or 425d37b6420cb7c4cc0c50af4a55684685525f0f

**Comment by hermunn** [comment]
Backport review: H2 is not in Varnish 4.1, so there is nothing to backport here.
